reference/api/settings.mdx

```diff
@@ -1910,3 +1911,333 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
+
+Use this field to use a specific revision of a model.
+
+This field is optional for the `huggingFace` embedder.
+
+This field is incompatible with all other embedders.
+
+##### `inputField`
+
+Indicates the path to the field containing document data in the query Meilisearch sends to the embedder.
+
+This field must be an array of strings. Each string should indicate a field within the request object. For example, if `inputField` is `["A", "B", "C"]`, the document data should be in `request.A.B.C`.
+
+This field is optional for the `rest` embedder.
+
+This field is incompatible with all other embedders.
+
+##### `inputType`
+
+Defines how Meilisearch sends the data to the embedder.
+
+`text`: Meilisearch submits a single request per document, and receives a single embedding as a response. When using `text`, you must specify the path to the embedding response in `embeddingObject`.
+
+`textArray`: Meilisearch submits multiple documents in a single request, and receives the same number of embeddings as a response. When using `textArray`, you must specify the path to the array of results in `pathToEmbeddings`, then the path from each item of the array to the embedding in `embeddingObject`.
+
+`textArray` results in fewer requests to the embedder and is the preferred `inputType`. However, not all embedders support sending multiple documents in a single request.
+
+This field is optional for the `rest` embedder.
+
+This field is incompatible with all other embedders.
+
+##### `pathToEmbeddings`
+
+Array indicating the path to the field containing all vector embeddings in the embedder response.
+
+This field must be an array of strings. Each string should indicate a field within the request object. If `inputType` is `text`, the last string in the array should indicate an array of numbers with the same dimensions as the embedding models. If `inputType` is `textArray`, the last string in the array should indicate an array of objects, each of which corresponds to a single document in the original request.
+
+For example, if `pathToEmbeddings` is `["A", "B", "C"]`, the document data should be in `response.A.B.C`.
+
+This field is optional for the `rest` embedder.
+
+This field is incompatible with all other embedders.
+
+##### `embeddingObject`
+
+Array indicating the path to the field containing a single set of vector embedding in a `textArray` embedder response.
+
+This field must be an array of strings. Each string should indicate the field containing vector data for a single element when the embedder returns multiple embeddings.
+
+For example, if `pathToEmbeddings` is `["A", "B", "C"]`, the document data should be in `response.A.B.C`, where `C` is an array of objects. For each element in `C`, Meilisearch will look for the path configured in `embeddingObject`. If `embeddingObject` is `["D", "E", "F"]`, vector data should be in `element.D.E.F`.
+
+This field is optional for the `rest` embedder. It is mandatory when `inputType` is `textArray`.
```

Comment:
```suggestion
This field is optional for the `rest` embedder.
```

The field is always optional and defaults to `[]`. 

In the case where `inputType` is `textArray` and `pathToEmbeddings` is `["A", "B", "C"]`, then the document data is in `response.A.B.C`, where `C` is an array of objects.

When `embeddingObject == []` in this situation, then each item of the array `C` is directly an embedding (that is, an array of floats).